python/lint/rules.bzl

```diff
@@ -49,7 +50,7 @@ def add_python_lint_tests(pylint = True, rcfile = None, **kwargs):
         pylint_test(
             name = "pylint",
             srcs = depset(pylint_srcs).to_list(),
-            deps = depset(transitive = [depset(pylint_deps)]),
+            deps = depset(direct = deps, transitive = [depset(pylint_deps)]),
```

Comment:
I'm not actually using this since the dep is added internally in the `pylint_test` rule, but it should allow us to add deps from an external call if we want to in the future.

Reply:
man, if we ever add a second linter here we'll have a bad time trying to figure out which args apply to which... I guess that's a future problem

Reply:
True, maybe we'd want a struct or dict like
```py
add_python_lint_tests(
  test_args = {
    "pylint": {"deps": [...]},
    "mypy": {"args": [...]},
   }
)
```